libobs/obs-output.c

```diff
@@ -290,6 +290,7 @@ void obs_output_destroy(obs_output_t *output)
 			if (output->video_encoders[i]) {
 				obs_encoder_remove_output(
 					output->video_encoders[i], output);
+				obs_encoder_release(output->video_encoders[i]);
```

Comment:
should this also do `output->video_encoders[i] = NULL;`? doesn't seem like there's any signal processing after this, and `obs_encoder_remove_output`/`destroy_caption_track` shouldn't try to access `output->video_encoders` either, so it's probably fine as is

Reply:
The entire struct gets free'd at the end of this function so I'd hope that at this point there's nothing trying to access the encoders at least